llvm/lib/Transforms/Yk/ControlPoint.cpp

```diff
@@ -104,13 +104,12 @@ class YkControlPoint : public ModulePass {
 
   bool runOnModule(Module &M) override {
     LLVMContext &Context = M.getContext();
-
     // Locate the "dummy" control point provided by the user.
     CallInst *OldCtrlPointCall = findControlPointCall(M);
     if (OldCtrlPointCall == nullptr) {
-      Context.emitError(
-          "ykllvm couldn't find the call to `yk_mt_control_point()`");
-      return false;
+      // This program doesn't have a control point. We can't do any
+      // transformations on it, but we do still want to compile it.
+      return true;
```

Comment:
This return should stay "false". It tell the compiler that this pass hasn't made any changes to the module, which is the case here.

Reply:
Fixed in 72a0a553ed4.